localstack/services/cloudformation/engine/template_preparer.py

```diff
@@ -37,9 +40,15 @@ def template_to_json(template: str) -> str:
     return json.dumps(template)
 
 
+# TODO: consider moving to transformers.py as well
 def transform_template(template: dict, parameters: list) -> Dict:
     result = dict(template)
 
+    # apply 'Fn::Transform' intrinsic functions (note: needs to be applied before global
+    #  transforms below, as some utils - incl samtransformer - expect them to be resolved already)
+    result = apply_transform_intrinsic_functions(result)
```

Comment:
That might turn out to be a bit of a problem. As far as I understand this is actually the opposite of what CloudFormation does.

Reply:
Agreed, had a similar thought. We'll see how it goes over time, especially as we further increase the surface area of snapshot tests. If this is not in line with AWS, we might end up having to perform some patches in samtransformer or other transformer libraries, to be able to handle `Fn::Transform` intrinsic functions within the templates. 👍